src/mapping/RadialBasisFctMapping.hpp

```diff
@@ -11,7 +11,7 @@
 #include "precice/types.hpp"
 #include "query/Index.hpp"
 #include "utils/EigenHelperFunctions.hpp"
-#include "utils/Event.hpp"
+#include "../EventTimings/include/EventTimings/Event.hpp"
```

Comment:
It feels strange that we need to set a specific path. We should rather add this to the include path.

(same in the `.cpp` file)